lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -505,12 +505,12 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
 
     onImageLoadingError(event: Event, row: DataRow) {
         if (event) {
-            let element = <any> event.target;
+            let element = <any>event.target;
 
             if (this.fallbackThumbnail) {
                 element.src = this.fallbackThumbnail;
             } else {
-                element.src = row.imageErrorResolver(event);
+                element.src = '//:0';
```

Comment:
The imageErrorResolver duplicates the icon of the first column into the blockchain state column. We just want a blank image when the state is not set.